cmd/collectors/rest/rest.go

```diff
@@ -502,6 +502,7 @@ func (r *Rest) HandleResults(result []gjson.Result, prop *prop, isEndPoint bool)
 						labelString := r.String()
 						labelArray = append(labelArray, labelString)
 					}
+					sort.Strings(labelArray)
```

Comment:
As discussed in short call, This sorting changes won't be impacting the `applications` labels mentioned in above security account plugin.

But this https://github.com/NetApp/harvest/blob/main/conf/rest/9.12.0/netroute.yaml#L11 can be impacted. 
`Ipaddress` and `names` are co-related. Checking this to avoid dependency of the labels

Reply:
Handled dependent labels in plugin by parsing whole field in template and parsed required labels in plugin.

Reply:
Changed and handled in template itself with gjson